schema_parse.go

```diff
@@ -485,19 +505,41 @@ func parseFixed(namespace string, m map[string]any, seen seenCache, cache *Schem
 	return fixed, nil
 }
 
-func parseFixedLogicalType(size int, lt string, prec, scale int) LogicalSchema {
+func parseFixedLogicalType(size int, lt string, otherProps map[string]any) LogicalSchema {
```

Comment:
Any reason not to name `otherProps` `props` instead? I am not sure the `other` brings much value.

Reply:
Okay, will change.

Reply:
Fixed